homeassistant/components/benq_projector/LICENSE

```diff
@@ -0,0 +1,201 @@
+                                 Apache License
```

Comment:
This should not be part of an integration, the project already has a license, please remove this file.